src/jsxc.lib.xmpp.chatState.js

```diff
@@ -130,9 +130,10 @@ jsxc.xmpp.chatState.onPaused = function(ev, jid) {
    }
 
    if (usersComposing.length === 0) {
-      var duration = parseFloat(el.css('transition-duration'));
+      var durationValue = el.css('transition-duration') || '0';
+      var duration = parseFloat(durationValue) || 0;
 
-      if (el.css('transition-duration').match(/s$/)) {
+      if (durationValue.match(/s$/)) {
```

Comment:
I don't know, what values you expect anyway, but as I understand the above mentioned spec, there's also the option of "ms". So maybe: `.match(/[^m]s$/)`?